src/test/java/racinggame/domain/CarFactoryTest.java

```diff
@@ -0,0 +1,50 @@
+package racinggame.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static racinggame.domain.CarFactory.createCars;
+
+public class CarFactoryTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"pobi,crong,pobi"})
+    @DisplayName("각 자동차의 이름은 중복될 수 없다")
+    void carNamesCannotBeDuplicated(String names) {
+        assertThatThrownBy(() -> createCars(names)).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"pobi,crong,honux"})
+    @DisplayName("자동차의 이름은 쉼표(,)를 기준으로 구분한다")
+    void carNamesAreSeparatedByComma(String names) {
+        List<Car> cars = createCars(names);
+
+        assertThat(cars).hasSize(3);
+        assertThat(cars.get(0).getName()).isEqualTo("pobi");
+        assertThat(cars.get(1).getName()).isEqualTo("crong");
+        assertThat(cars.get(2).getName()).isEqualTo("honux");
```

Comment:
```suggestion
        List<Car> cars = createCars(names);

        assertThat(cars).hasSize(3);
        assertThat(cars).cotains(new Car("pobi"), new Car("crong"), new Car('honux"));
```
위와 같이 객체의 값을 꺼내 비교하기보다 객체 단위로 비교하는 방식으로 접근할 것을 추천